### ISSUE TITLE
Thanks for sharing your code. It's really a nice work!

### ISSUE THREAD

**Issue by ironshangki** [Calamari-OCR/calamari#121]
During training, I met the error:
tensorflow/core/util/ctc/ctc_loss_calculator.cc:144] No valid path found.

However, the loss decreased normally and after training the prediction of input line images are very similar to the ground truth .

The command I used is: 
calamari-train --files train_img_path.files --batch_size 64 --display 1 --max_iters 100000 
--output_dir iam_models --checkpoint_frequency 1000 --no_skip_invalid_gt

I really want to know how to solve it. Thanks a lot!

**Comment by ChWick** [comment]
Duplicate of #66 and #89 

You can ignore it if the model trains successfully.
